### PR TITLE
Porting persistant_simctrl test bucket to open liberty 

### PR DESCRIPTION
**Pull request by KyleAure**
Commit History
Original port
Refactoring to work in OL

### CODE REVIEW

**Comment by KyleAure** [comment]
#build

File: dev/com.ibm.ws.concurrent.persistent_fat_simctrl/build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+
+addRequiredLibraries.dependsOn addDerby
+
+dependencies {
+  requiredLibs project(':com.ibm.ws.concurrent.persistent_fat_simctrl')
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent.persistent_fat_simctrl/build.gradle]: this dependency shouldn't be needed -- the FAT bundle gets added to the project classpath by default

File: dev/com.ibm.ws.concurrent.persistent_fat_simctrl/publish/files/features/controllerTestFeature-1.0.mf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Subsystem-ManifestVersion: 1
+IBM-ShortName: controllerTestFeature-1.0
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent.persistent_fat_simctrl/publish/files/features/controllerTestFeature-1.0.mf]: This feature file should have the `IBM-Test-Feature: true` line in it somewhere to indicate that it's a test feature which acts as a safety net to make sure we never accidentally ship it

File: dev/com.ibm.ws.concurrent.persistent_fat_simctrl/publish/files/features/simulatedController-1.0.mf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Subsystem-ManifestVersion: 1
+IBM-ShortName: simulatedController-1.0
```
Comment (aguibert) [inline dev/com.ibm.ws.concurrent.persistent_fat_simctrl/publish/files/features/simulatedController-1.0.mf]: same comment here about the `IBM-Test-Feature` line

**Comment by KyleAure** [comment]
#build

**Pull request merged**